src/api/app/models/staging/workflow.rb

```diff
@@ -28,11 +28,7 @@ def ready_to_stage
   after_create :create_staging_projects
 
   def unassigned_requests
-    target_of_bs_requests.stageable - staged_requests - ignored_requests
-  end
-
-  def ready_requests
-    target_of_bs_requests.ready_to_stage - staged_requests - ignored_requests
+    target_of_bs_requests.in_states(['new', 'review']).where.not(id: staged_requests.pluck(:id)).where.not(id: ignored_requests.pluck(:id))
```

Comment:
Workaround, can be dropped when https://github.com/openSUSE/open-build-service/pull/6257 is merged